### ISSUE TITLE
Ability to contribute data without inbound connection/port forward

### ISSUE THREAD

**Issue by Romeo-Golf** [openskynetwork/raspberry-pi-adsb#1]
I suspect that the need for a user to open ports on their firewall may deter some people from contributing... Is there any plans to allow users to submit beast format data by forwarding it to your servers, as the majority of other feeders typically operate?

**Comment by affitz** [comment]
Yes! In fact we are about to release a push feeder for beast format data in January. We are just doing some final testing.

**Comment by Romeo-Golf** [comment]
Sounds great, I'm happy to help testing this and once released work to integrate this into the adsbreceiver project :)

**Comment by highsource** [comment]
Does adsb-receiver work on Raspberri Pi?

**Comment by Romeo-Golf** [comment]
The ADS-B Receiver Project ✈️

This repository contains a set of scripts and files which can be used to setup an ADS-B receiver on a clean installation of a Debian derived operating system. The scripts are executed in order by the main install script depending on the installation options chosen by the user.

**Comment by Romeo-Golf** [comment]
And yes, it's largely targeted at the Raspberry Pi platform with a prebuilt Raspian Jessie image that allows installation of dump1090 for collection of ADS-B data from a RTL-SDR device and the ability to view this on a map.

The install script currently providers the option to configure export/sharing of the received data with Flightaware, FlightRadar24, Plane Finder and AdsbExchange... But it should be easy enough to add support for the OpenSkyNetwork once this project is able to accept data pushed in the BEAST format exported by dump1090.

There is a web portal for configuration, statistics and links to the local or cloud based clients that accompany beach feeder :)

Hope that helps.

**Comment by Romeo-Golf** [comment]
Hi, any news on being able to accept beast format data from users who are not able to open up their firewalls?

**Comment by affitz** [comment]
Feeder finished and released. Find the installation instructions here: https://opensky-network.org/community/projects/30-dump1090-feeder

@Romeo-Golf Feel free to integrate it into the adsbreceiver.net project. We're happy to help if there are questions.

**Comment by highsource** [comment]
I have to document it here - please leave open.

**Comment by Romeo-Golf** [comment]
Thanks for the heads up, I now have this feeder running on 3 Pi's with no significant issues so should be straight forward to integrate into the adsb-receiver :)

**Comment by aggy0** [comment]
hello people
